src/abhishekSJun20/FirstDifferIndexOfArray.java

```diff
@@ -0,0 +1,25 @@
+package abhishekSJun20;
+/*
+ * Program 2: Find first differ index from given arrays. 
+                   i/p :- arr1: {10,2,9,14,3}
+                            arr2: {10,2,18,14,3}
+                   o/p:- Values are not matching at index -> 
+ */
+
+public class FirstDifferIndexOfArray {
+
+	void differIndexValueInIndex(int[] arr1, int[] arr2) {
+
+		for (int index = 0; index < arr1.length; index++) {
+			if (arr1[index] != arr2[index])
+				System.out.println("Values are not matching at index -> " + index);
+		}
```

Comment:
Here you are checking for the first index, why return statement is missing ??

Reply:
done changes as per comment.